src/code/templates/fv-egg-game.js

```diff
@@ -485,15 +484,8 @@ var animationEvents = {
                         targetPositionRect: getDstChromBounds(i), endSize: 0.2 });
         ++animationEvents.moveChromosomesToGamete.activeCount;
       }
-      if (!debugSkipRandomGameteAnimation) {
-        animateMultipleComponents(components, positions, opacity, speed,
-                                  animationEvents.moveChromosomesToGamete.id,
-                                  animationEvents.moveChromosomesToGamete.onFinish);
-      }
-      else {
-        for (let i = animationEvents.moveChromosomesToGamete.activeCount; i >= 0; --i) {
-          animationEvents.moveChromosomesToGamete.onFinish(animationEvents.moveChromosomesToGamete.id);
-        }
+      for (let i = animationEvents.moveChromosomesToGamete.activeCount; i >= 0; --i) {
```

Comment:
We removed the animation where the chromosomes move to the gamete to speed things up